features/attach_validtoken.feature

```diff
@@ -143,28 +143,21 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         Then if `<release>` in `xenial` and stdout matches regexp:
         """
 
-        \*Your UA Infra: ESM subscription has EXPIRED\*
+        \*Your Ubuntu Pro: ESM Infra subscription has EXPIRED\*
 
-        \d+ additional security update\(s\) could have been applied via UA Infra: ESM.
+        \d+ additional security update\(s\) could have been applied via Ubuntu Pro: ESM Infra.
 
-        Renew your UA services at https:\/\/ubuntu.com\/advantage
-
-        """
-        Then if `<release>` in `xenial` and stdout matches regexp:
-        """
-
-        Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
-        applicable law.
+        Renew your UA services at https:\/\/ubuntu.com\/pro
 
         """
         When I run `apt upgrade --dry-run` with sudo
         Then if `<release>` in `xenial` and stdout matches regexp:
         """
-        \*Your UA Infra: ESM subscription has EXPIRED\*
-        Enabling UA Infra: ESM service would provide security updates for following packages:
+        \*Your Ubuntu Pro: ESM Infra subscription has EXPIRED\*
+        Enabling Ubuntu Pro: ESM Infra service would provide security updates for following packages:
           .*
         \d+ esm-infra security update\(s\) NOT APPLIED. Renew your UA services at
-        https:\/\/ubuntu.com\/advantage
+        https:\/\/ubuntu.com\/pro
```

Comment:
Do we want to make this change yet? @lucasmoura
We might release before the URL is live, in which case there will be a few days with a broken link - but maybe that's not that bad?